src/containers/BooksForm.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+
+const styles = {
+  input: {
+    marginRight: "10px",
+    height: "45px",
+    width: "60%",
+    fontSize: "18px"
+  },
+  select: {
+    marginRight: "10px",
+    height: "50px",
+    width: "20%",
+    fontSize: "18px"
+  },
+  button: {
+    padding: "10px",
+    width: "18%",
+    background: "#0290ff",
+    color: "#fff",
+    padding: "16px",
+    fontSize: "18px"
+  }
+}
+
+
+const bookCategories = [
```

Comment:
`booksCategories` occurs here in `BooksForm` as well as in `CategoryFilter`. Let us remove duplication and just house this array in a separate file in a `utils` folder.